### PR TITLE
gh-153: First simple e2e test

### PR DESCRIPTION
**Pull request by littlewhywhat**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,8 +5,10 @@
   "main": "index.js",
   "scripts": {
     "start": "cd backend && npm run create-schema && cd .. && ./start.sh",
-    "postinstall": "cd backend && npm install && cd ../frontend && npm install && npm run build && cd ..",
-    "lint": "eslint --max-warnings 0 './backend/**/*.ts' './frontend/src/**/*.js' --ignore-pattern '/backend/node_modules/*' --ignore-pattern '/frontend/node_modules/*'"
+    "lint": "eslint --max-warnings 0 './backend/**/*.ts' './frontend/src/**/*.js' --ignore-pattern '/backend/node_modules/*' --ignore-pattern '/frontend/node_modules/*'",
```
Comment (novellizator) [inline package.json:8]: can't you simply ignore the node_modules altogether without listing the out in both directories?

File: package.json [hunk 2]
```diff
@@ -5,8 +5,10 @@
...
+    "start:e2e": "concurrently \"cd backend && npm start\" \"wait-on http://localhost:3000/games && cd frontend && npm run start:e2e\"",
```
Comment (novellizator) [inline package.json:9]: do we need to specifically wait on "/games" endpoint, or is "/" enough?

File: .github/workflows/e2e.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+jobs:
+  e2e:
+    runs-on: ubuntu-latest
+    # cypress requires xvfb
+    container: thompsnm/nodejs-chrome-xvfb
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: XXXX
```
Comment (novellizator) [inline .github/workflows/e2e.yml:18]: i might make the user and db explicit (POSTGRES_USER, POSTGRES_DB)

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -17,10 +17,12 @@ jobs:
     - name: FE unit tests
       run: |
         cd frontend
+        npm install
```
Comment (novellizator) [inline .github/workflows/build.yml:20]: you might want to take a look at this https://docs.npmjs.com/cli/ci.html

File: package.json (hunk 1 above)
Comment (littlewhywhat) [reply package.json]: true, thank you, fixed here https://github.com/salsita/foosball-rating/pull/154/commits/6de4130a5334e3f7cc77b41a9f0f53a38de58ed2

File: package.json (hunk 2 above)
Comment (littlewhywhat) [reply package.json:9]: wait-on waits for 2xx on GET and for "/" we give 404 only right now.

File: .github/workflows/build.yml (hunk 4 above)
Comment (littlewhywhat) [reply .github/workflows/build.yml:20]: thanks, fixed here https://github.com/salsita/foosball-rating/pull/154/commits/cd851298d2d5058b4f9b94c5e69b4e63ea390d0e

File: .github/workflows/e2e.yml (hunk 3 above)
Comment (littlewhywhat) [reply .github/workflows/e2e.yml:20]: ok, fixed here https://github.com/salsita/foosball-rating/pull/154/commits/31a8ba5da60da761aea1132031a4890dfdba22cd

File: .github/workflows/build.yml (hunk 4 above)
Comment (littlewhywhat) [reply .github/workflows/build.yml]: and here :) https://github.com/salsita/foosball-rating/pull/154/commits/dfec6545ed6d9a98ad73fbd0ee640deb8ae7d52a

File: package.json (hunk 2 above)
Comment (novellizator) [reply package.json:9]: a pity we cannot possibly make comments on this in a .json file. I'd think / should return a 200 in order to have a healthcheck implementable on it. in it's simplest form `{alive: true}` should be a sufficient response. Let's leave it for the future

**Pull request merged**